test/headful.spec.js

```diff
@@ -116,6 +117,13 @@ module.exports.addTests = function({testRunner, expect, puppeteer, defaultBrowse
       await page.click('body');
       await browser.close();
     });
+    it('should open devtools when "devtools: true" option is given', async({server}) => {
```

Comment:
firefox won't like this

Reply:
Why? We run headful tests on CHROME only